sdk/schemaregistry/azure-data-schemaregistry-avro/src/main/java/com/azure/data/schemaregistry/avro/SchemaRegistryAvroSerializer.java

```diff
@@ -45,30 +45,55 @@
         this.autoRegisterSchemas = autoRegisterSchemas;
     }
 
+    /**
+     * Deserializes the {@code inputStream} into a strongly-typed object.
+     *
+     * @param inputStream The stream to read from.
+     * @param typeReference Type reference of the strongly-typed object.
+     * @param <T> Strongly typed object.
+     *
+     * @return The deserialized object. If {@code inputStream} is null then {@code null} is returned.
+     *
+     * @throws NullPointerException if {@code typeReference} is null.
+     */
```

Comment:
Is this required? Can this just be inherited from the parent javadoc?

Reply:
I wanted to clarify the behaviour when typeReference or inputStream was null because the docs don't explain it